test/parse.js

```diff
@@ -843,3 +843,53 @@ test('parse()', function (t) {
 
     t.end();
 });
+
+test('parse/stringify empty key WIP', function (t) {
+    t.test('parses an object with empty string key', function (st) {
+        st.deepEqual(qs.parse('=a', { allowEmptyKeys: true }), { '': 'a' });
```

Comment:
for each of these test inputs, let's test both allowEmptyKeys true, false, and absent

Reply:
Done, please let me know what will be the correct place for this tests? They are testing both parse and stringify and current tests are split by this methods. To me it feels more reasonable to test them together in this case.